js/pages/tola_management_pages/country/models.js

```diff
@@ -344,10 +333,11 @@ export class CountryStore {
                     this.updateHistory(id);
                 }))
             .catch((errors) => {
+                console.log(errors);
```

Comment:
Left in on purpose?

Reply:
whoops, no - one-line pr coming right up